subcommittees/policy-CoC/README.md

```diff
@@ -0,0 +1,33 @@
+# The Code of Conduct Policy Subcommittee
+
+## Purpose and Scope
+
+Enforce the Code of Conduct (CoC) in all parts of the Carpentries community
+
+## What We Do
+
+* communicate with the Carpentries community about the CoC, as well as reporting and enforcement guidelines
+* review and respond to reports of CoC violations as deemed appropriate by the situation
+
+## Contact
+
+* policy@carpentries.org
+* confidential@carpentries.org
+
+## Current Members
+
+* Pauline Barmby - chair (community)
+* Chris Hamm (community)
+* Simon Waldman (community)
+* Erin Becker (staff)
+* Jonah Duckles (staff)
+
+## Past Members
+
+We don't have any past members yet!  When current members leave the
+Subcommittee we will move their entries into this section.
+
+[blog]: https://software-carpentry.org/blog/
+[blog-archives]: https://software-carpentry.org/blog/categories/#SLUG
+[mailing-list]: http://lists.software-carpentry.org/listinfo/SLUG
+[mailing-list-archives]: http://lists.software-carpentry.org/pipermail/SLUG/
```

Comment:
I tried to clarify this a bit, but am not on the committee so not sure what their intentions are.